packages/backend/src/functions/socketServer/SocketServer.ts

```diff
@@ -144,9 +145,9 @@ export default class SocketServer {
     return this.userMap.get(socketId)?.getUserData();
   }
 
-  public destroyUser(login: string): void;
-  public destroyUser(user: User): void;
-  public destroyUser(user: User | string): void {
+  public destroyUser(login: string, clearSession: boolean): void;
+  public destroyUser(user: User, clearSession: boolean): void;
+  public destroyUser(user: User | string, clearSession = true): void {
```

Comment:
Cant find any occurrence when this is called with a string, is this old stuff?.. The middle function declaration should suffice?

Reply:
Both of what you say is true!

Yes, the string argument is never used and yes, this is old stuff